docs/features/onboarding.md

```diff
@@ -19,35 +19,107 @@ You can also set the following configuration flag in your `wp-config.php`:
+* POST `/wc-admin/onboarding/tasks/import_sample_products` - Used for importing sample products during the appearance task.
+* POST `/wc-admin/onboarding/tasks/create_store_pages` - Used for creating default store pages (like my account and checkout) during the profile wizard.
+* POST `/wc-admin/onboarding/tasks/create_homepage` - Used for creating a homepage using Gutenberg templates.
+
+## Onboarding filters
+
+* `woocommerce_onboarding_profile_properties` filters the properties we track as part of the profile wizard (such as information from store/business details steps).
+* `woocommerce_rest_onboarding_profile_object_query` filters the query arguments for requests to `/wc-admin/onboarding/profile`.
+* `woocommerce_rest_prepare_onboarding_profile` filters the response for requests to `/wc-admin/onboarding/profile`.
+* `rest_onboarding_profile_collection_params` filters the collection parameters for requests to  `/wc-admin/onboarding/profile`.
+* `woocommerce_admin_onboarding_industries` filters the list of allowed industries displayed in the profile wizard.
+* `woocommerce_admin_onboarding_industry_image` filters the images used for homepage templates in the appearance task. When creating a homepage, example images are used based on industry. These images are stored in `images/onboarding`.
+* `woocommerce_admin_onboarding_product_types` filters the product types displayed in the profile wizard.
+* `woocommerce_onboarding_plugins_whitelist` filters the list of plugins that can installed & activated via onboarding. This acts as a whitelist so only certain plugins can be used via the `/wc-admin/onboarding/profile/install` and `/wc-admin/onboarding/profile/activate` endpoints.
+* `woocommerce_admin_onboarding_themes` filters the themes displayed in the profile wizard.
+* `woocommerce_onboarding_jetpack_connect_redirect_url` filters the Jetpack connection URL outlined in the Jetpack connection section below.
+* `woocommerce_onboarding_task_list` filters the list of tasks on the task list dashboard. This allows extensions to add new tasks. See [the extension docs](https://github.com/woocommerce/woocommerce-admin/tree/42015d17a919e8f9e54ba75869c50b04b8dc9241/docs/examples/extensions) for an example of how to do this.
 
-## Onboarding filters and hooks
-
-@todo
 
 ## Options and settings
 
-@todo
+A few new WordPress options have been introduced to store information and settings during setup. It may be necessary to manual delete these options from your `wp_options` database to test a certain task or feature.
 
-## WooCommerce.com connection and redirection
+* `woocommerce_onboarding_payments`. Since the payments step requires multiple redirects to payment providers to setup accounts, we cache the current progress of the payments step in an option, so that we can quickly drop users back into the correct part of the task.
```

Comment:
We can also add `woocommerce_task_list_hidden` to this list.

Reply:
Thanks for catching that -- definitely one of the more important ones to know about!